CMakeSettings.json

```diff
@@ -0,0 +1,15 @@
+﻿{
```

Comment:
what is this?

Reply:
probably can remove. it is used by MS Visual Studio's built-in CMake support.